src/services/admin/email.service.ts

```diff
@@ -0,0 +1,64 @@
+import { dataSource } from '../../configs/dbConfig'
+import Email from '../../entities/email.entity'
+import type { EmailStatusTypes } from '../../enums'
+
+export const createEmailTemplate = async (
+  recipient: string,
+  subject: string,
+  content: string,
+  state: EmailStatusTypes
```

Comment:
Create email template only requires the content and subject. A sent email will have the data you mentioned above

Reply:
Noted Thanks